code/src/interfaces/parseResponse.ts

```diff
@@ -0,0 +1,65 @@
+    export interface Guild {
+        id?: number;
+        name: string;
+        faction?: number;
```

Comment:
is there a limited number of factions possible ? if so you could do something like `type Faction = 1 | 2 | 56`